config/crd/bases/agent-install.openshift.io_agents.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.6.2
```

Comment:
no need for these changes in this pr...

Reply:
it's temporary problem in master that will be solved soon